src/main/java/com/splunk/hecclient/JsonEventBatch.java

```diff
@@ -43,4 +46,23 @@ public String getContentType() {
     public EventBatch createFromThis() {
         return new JsonEventBatch();
     }
-}
+
+    @Override
+    public int hashCode() {
+        return new HashCodeBuilder()
+        .append(endpoint)
+        .toHashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj instanceof JsonEventBatch) {
+            final JsonEventBatch other = (JsonEventBatch) obj;
+            return new EqualsBuilder()
+            .append(endpoint, other.endpoint)
+            .isEquals();
+        } else {
+        return false;
```

Comment:
we can get ride of `else` and just `return false` directly ?
and essentially the equals compares if endpoint is the same ? is this safe for putting EventBatch in a hashtable ?

Reply:
Modified return.
Its a helper function to be used to help make batching of events better for the /raw endpoint. 
It is safe and without issue.